docs/UserGuide.md

```diff
@@ -159,7 +159,10 @@ Example:
 
 ### Retrieving payroll `payroll`
 
-Coming soon!
+Retrieve employee's payroll for a given start and end date
+
```

Comment:
Maybe add the format before the example as well! otherwise lgtm

Reply:
donez